COMET.Web.Common/Components/MultiComboBox.razor.cs

```diff
@@ -0,0 +1,97 @@
+﻿// -----------------------------------------------------------------------------------
+//   <copyright file="MultiComboBox.razor.cs" company="RHEA System S.A.">
+//      Copyright (c) 2023 RHEA System S.A.
+// 
+//      Authors: Sam Gerené, Jaime Bernar

```

Comment:
This component was copied from another project called AIDA. Please align the headers with the headers for the WebApp